scripts/superdesk-search/search.js

```diff
@@ -2248,9 +2248,18 @@
          * Multiple item spike
          */
         this.spikeItems = function() {
-            spike.spikeMultiple(multi.getItems());
-            $rootScope.$broadcast('item:spike');
-            multi.reset();
+            if ($location.path() === '/workspace/personal') {
+                modal.confirm(gettext('Do you want to delete the item permanently? '))
```

Comment:
Since these are multiple items, the more appropriate text would be _"Do you want to delete these items permanently?"_

Also, I might be wrong, but as far as I know these items can be unspiked, meaning that the action is not permanent as the confirmation dialog says.